### PR TITLE
LPS-167258 [CP 2.0] The 'All' option has to be visible in the dropdown

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

**Comment by luisacmn** [comment]
ci:test:sf

**Comment by luisacmn** [comment]
ci:test:osb-site-initializer-customer-portal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/SubscriptionsNavbar/components/SubscriptionStatusDropdown/SubscriptionStatusDropdown.js [hunk 1]
```diff
@@ -62,18 +62,53 @@ const SubscriptionStatusDropdown = ({disabled, loading, onClick}) => {
 		setItems([...items]);
 	};
 
-	const getDropdownItems = () =>
-		items.map((item, index) => (
+	const handleClickAll = () => {
+		items.active = !items.active;
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/SubscriptionsNavbar/components/SubscriptionStatusDropdown/SubscriptionStatusDropdown.js:66]:
```suggestion
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/SubscriptionsNavbar/components/SubscriptionStatusDropdown/SubscriptionStatusDropdown.js [hunk 2]
```diff
@@ -62,18 +62,53 @@ const SubscriptionStatusDropdown = ({disabled, loading, onClick}) => {
...
+		});
+
+		setItems([...newItems]);
+		const currentActiveItems = items.filter((item) => item.active);
+		if (currentActiveItems.length === MAX_SUBSCRIPTION_STATUS) {
+			onClick(currentActiveItems.map((item) => item.label));
+		}
+		else {
+			onClick();
+		}
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/SubscriptionsNavbar/components/SubscriptionStatusDropdown/SubscriptionStatusDropdown.js:79]:
```suggestion
		onClick(items.map((item) => item.label));
```

Comment (luisacmn) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/SubscriptionsNavbar/components/SubscriptionStatusDropdown/SubscriptionStatusDropdown.js]: Alterado!

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**